README.md

```diff
@@ -1 +1,145 @@
+
+## 🛠️ Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- <a href="https://www.ruby-lang.org/es/">ROR</a>
+- <a href="https://www.postgresql.org/">Postgresql</a>
+
+### Key Features <a name="key-features"></a>
+
+- ORM model
+- User, Comment, Post, and like model
+- Custom methods in each classes
+- Add validation for user and post
+- Test cases for each model
+- Add views, controllers, and routes for each model
+- Test cases for each controller
+- Add views for user and posts
+- Add forms for posts, comments, and likes
+- Add integtation test and fix n+1 problems
+- Add authentication using devise gem
+- Add authorization using cancancan gem
+- Add API endpoint
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Command line or terminal
+- IDE such as VsCode.
+- Rails needs to be installed on your system.
+
+### Setup
+
+Clone this repository to your desired folder:
+Example command:
+
+```sh
+  git clone https://github.com/Lornakaboro/rails-blog.git
+  cd blog-app
+
+```
+
+### Install
+
+Example command:
+
+```sh
+  cd blog_app
+  bundle install (install all dependencies)
+
+```
+
+### Usage
+
+To run the project redirecting to the root directory of the project and run the following command:
+
+```sh
+ bin/rails server (to run web application)
+
+```
+
+### Test
+
+To test the web app run the following command:
+
+```sh
+ rspec spec ( to test the web application)
+
+```
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Lornakaboro**
+
+- GitHub: [@githubhandle](https://github.com/Lornakaboro)
+- Twitter: [@twitterhandle](https://twitter.com/KaboroLorna)
+- Linkedin [@linkedinprofile](https://www.linkedin.com/in/lorna-kaboro-23620b242/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [add API documentation]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Lornakaboro/rails-blog/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐ Show your support <a name="support"></a>
+
+Give a ⭐ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+we would like to thank microverse community for the support and guidance.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
```

Comment:
- [ ] Notice the link for your MIT file is broken, Please make sure to add the MIT license file and update its content in regard to your name and project. Also replace the `[YOUR NAME]` part with your name, and `[APP TYPE]` with your app type.